compiler/rustc_ast/src/ast.rs

```diff
@@ -1446,6 +1446,9 @@ pub enum ExprKind {
     /// with an optional value to be returned.
     Yeet(Option<P<Expr>>),
 
+    /// Bytes included via `include_bytes!`
+    IncludedBytes(Span, std::path::PathBuf, Lrc<[u8]>),
```

Comment:
```suggestion
    IncludedBytes(Lrc<[u8]>),
```
- `PathBuf` is no longer used
- `Span` is always the same as the expression span